Since Void implements usrmerge, /lib is a symlink to /usr/lib. musl's default,
here, makes it so the libraries under /usr/local/lib won't override the system
ones, which is not the desired behavior.

diff --git a/ldso/dynlink.c b/ldso/dynlink.c
index 5b9c8be4..f6532968 100644
--- a/ldso/dynlink.c
+++ b/ldso/dynlink.c
@@ -1095,7 +1095,7 @@ static struct dso *load_library(const char *name, struct dso *needed_by)
 					sys_path = "";
 				}
 			}
-			if (!sys_path) sys_path = "/lib:/usr/local/lib:/usr/lib";
+			if (!sys_path) sys_path = "/usr/local/lib:/usr/lib";
 			fd = path_open(name, sys_path, buf, sizeof buf);
 		}
 		pathname = buf;
